views/messages.ejs

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, max-scale=1.0">
+    <title>VITCMUN 19: E-Chit</title>
+    <link rel="stylesheet" href="../public/css/font-awesome-4.7.0/css/font-awesome.min.css">
+    <!-- Bulma Version 0.7.2-->
+    <link rel="stylesheet" href="../public/css/bulma.min.css" />
+    <link rel="stylesheet" type="text/css" href="../public/css/inbox.css">
+</head>
+
+<body>
+
+    <!-- START NAV -->
+    <nav class="navbar dark" role="navigation" aria-label="main navigation">
+        <div class="navbar-brand">
+            <!--Big Flag for Desktop Device-->
+            <a class="navbar-item is-hidden-mobile navbar-flag" href="#">
+                <img class="icon-top" src="../media/flag/india.png" width="137mm" height="137mm">
+            </a>
+            <!--Small Flag for Mobile Devices-->
+            <a class="navbar-item is-hidden-tablet" href="#">
+                <img class="icon-top" src="../media/flag/india.png" width="50mm" height="50mm">
+            </a>
+            <!--Council Logo for Mobile Devices-->
+            <div class="navbar-item is-logo is-hidden-tablet">
+                <a class="is-logo" href="#">
+                    <img class="icon-top" src="../media/council_logos/EU.JPG" width="120">
+                </a>
+            </div>
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navbarMenu" class="navbar-menu">

```

Comment:
Have you used the id or class anywhere? Or is it redundant. If so one of them should be removed.

Reply:
It is used for the Nurger menu in nav bar

Reply:
I think the class might be a bulma class?

Reply:
Nah the id is used by the JS action to change the burger menu icons etc.

Reply:
Okay then!